pom.xml

```diff
@@ -348,6 +349,16 @@
             <artifactId>maven-artifact</artifactId>
             <version>3.3.9</version>
         </dependency>
+        <dependency>
+            <groupId>com.github.docker-java</groupId>
+            <artifactId>docker-java</artifactId>
+            <version>2.2.3</version>
+        </dependency>
+        <dependency>
+            <groupId>org.codehaus.plexus</groupId>
+            <artifactId>plexus-utils</artifactId>
+            <version>3.0.22</version>
```

Comment:
@original-brownbear what is the reason e.g. for that change? I don't see it described in PR desc